src/functional.ts

```diff
@@ -136,6 +179,103 @@ export function functionalInterfaceFactory({
     }
   }
 
+  function convertToBlstPublicKeyArg(publicKey: PublicKeyArg): blst.PublicKeyArg {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    return publicKey instanceof PublicKey ? ((publicKey as any).value as blst.PublicKey) : publicKey;
+  }
+
+  function convertToBlstSignatureArg(signature: SignatureArg): blst.SignatureArg {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    return signature instanceof Signature ? ((signature as any).value as blst.Signature) : signature;
+  }
+
+  /**
+   * Verifies if signature is message signed with given public key.
+   * @param publicKey
```

Comment:
remove these @param if we don't have comments for them, same for below functions

Reply:
Updated.